shell/platform/linux/fl_dart_project.cc

```diff
@@ -96,45 +81,32 @@ static void fl_dart_project_init(FlDartProject* self) {}
 
 /**
  * fl_dart_project_new:
- * @assets_path: a file path, e.g. "build/assets"
- * @icu_data_path: a file path, e.g. "build/icudtl.dat"
+ * @path: a file path, e.g. "my_dart_project"
  *
- * Create a Flutter project.
+ * Create a Flutter project. The project path should contain the following
+ * top-level items:
+ * - icudtl.dat (provided as a resource by the Flutter tool)
+ * - flutter_assets (as built by the Flutter tool)
+ *
+ * The path can either be absolute, or relative to the directory containing the
```

Comment:
The "or relative" part isn't actually true currently. Do you want to go ahead and add that to this PR? You should be able to just copy [this logic](https://github.com/flutter/engine/blob/master/shell/platform/glfw/flutter_glfw.cc#L512) (removing the u8string stuff, since you don't need Windows compatibility)